src/modules/schema.js

```diff
@@ -123,6 +125,17 @@ export default class Schema {
    * @returns {Boolean}
    */
   static validateSchema(json) {
+    let jsonSchemaSpec;
+    if (json['$schema']) {
+      if (json['$schema'].startsWith('http://json-schema.org/draft-07/schema')) {
+        jsonSchemaSpec = JSONSchema07;
+      } else {
+        // TODO: Fetch json['$schema'] (using http client) and convert function to async
```

Comment:
was unaware this method required http calls, in this case it should be marked as async in impl spec too

Reply:
Yes